testament/important_packages.nim

```diff
@@ -164,8 +162,7 @@ pkg "ssostrings"
 pkg "stew"
 pkg "stint", "nim c stint.nim"
 pkg "strslice"
-pkg "strunicode", "nim c -r --mm:refc src/strunicode.nim",
-  preCmd = "nimble install -y normalize@#HEAD; nimble uninstall -y normalize; nimble install -y normalize@#HEAD"
+pkg "strunicode", "nimble uninstall -i -y normalize; nimble install -y normalize@#HEAD; nimble install --depsOnly -y; nim c -r --mm:refc src/strunicode.nim"
```

Comment:
I've asked @nitely to bump `normalize`'s version, in hopes this hack won't be needed anymore.